velox/common/memory/MemoryAllocator.cpp

```diff
@@ -415,7 +413,7 @@ void MemoryAllocator::useHugePages(
     const ContiguousAllocation& data,
     bool enable) {
 #ifdef linux
-  if (!FLAGS_velox_memory_use_hugepages) {
+  if (config::globalConfig.memoryUseHugepages) {
```

Comment:
Hi @majetideepak the logic for enabling hugepage seems changed here, is this intended? 
- previously if we set `FLAGS_velox_memory_use_hugepages` to `false`, then it will do a early return. if we set to `true` then it will try to do `madvise()`
- now if we set `config::globalConfig.memoryUseHugepages` to `false`, then it will do `madvise()`. if we need to disable hugepage then we need to set it to `true` 

Thanks, -yuan